wiki/Gameplay/Grade/en.md

```diff
@@ -1,55 +1,60 @@
 ---
-needs_cleanup: true
+tags:
+  - letter grade
+  - rank
+  - silver s
+  - sh
+  - xh
 ---
 
 # Grade
 
-<!-- TODO (cleanup tasks): this is missing a lead section, and a XH/X note in the taiko section -->
+*For other uses, see [Rank (disambiguation)](/wiki/Disambiguation/Rank).*
+
+**Grades** are a set of letters representing a player's performance on a given [beatmap](/wiki/Beatmap). They accompany the total [score](/wiki/Gameplay/Score) on the result screen and appear in all lists alongside scores.
+
+From lowest to highest, the possible grades are D, C, B, A, S, and SS. Silver S and SS grades are only achievable when using the [Hidden](/wiki/Game_modifier/Hidden), [Flashlight](/wiki/Game_modifier/Flashlight), or [Fade In](/wiki/Game_modifier/Fade_In) game modifiers.
 
 ## osu!
 
 | Grade | Condition |
 | :-: | :-- |
 | SS | 100% accuracy |
-| S | Over 90% 300s, less than 1% 50s and no misses |
+| S | Over 90% 300s, 1% or less 50s, and no misses |
```

Comment:
Could have also done "at most 1% 50s" but this works too I guess.

Reply:
both mean exactly the same thing, although you're right in that it would look neater